mpt/impl/smtp/cassandra/src/test/java/org/apache/james/mpt/smtp/host/CassandraJamesSmtpHostSystem.java

```diff
@@ -101,10 +104,16 @@ public InMemoryDNSService getInMemoryDnsService() {
         return inMemoryDNSService;
     }
 
-    protected GuiceJamesServer createJamesServer() {
-        return new GuiceJamesServer()
+    protected GuiceJamesServer createJamesServer() throws IOException {
```

Comment:
what about propagating it as a runtime instead of making all these method to throw IOException?

Reply:
Here we can simply throw an exception